### DISCUSSION TITLE
Convert UTC to local time zone.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [js-fns/date-fns#3336] [General]
In the moment js it is so easy to convert UTC to local date time. Like this.

```
const utcDate = '2023-02-02 08:29:54';

const result = moment.utc(utcDate).local().format('YYYY-MM-DD HH:mm:ss');
```
But in the date-fns it is too hard to get the same result.

**Comment by FibreFoX** [comment]
Is there still no better way in date-fns? Or am I missing something?
